packages/react-accordion/src/components/AccordionHeader/useAccordionHeaderStyles.ts

```diff
@@ -8,12 +8,14 @@ const useStyles = makeStyles({
   // TODO: this should be extracted to another package
   resetButton: {
     boxSizing: 'content-box',
-    background: 'none',
+    backgroundColor: 'inherit',
     color: 'inherit',
-    font: 'inherit',
+    fontFamily: 'inherit',
+    fontSize: 'inherit',
     lineHeight: 'normal',
-    overflow: 'visible',
-    padding: '0',
+    overflowX: 'visible',
+    overflowY: 'visible',
```

Comment:
don't  you have a shorthand for this  ?

```suggestion
    ...shorthands.overflow('visible'),
```

Reply:
Yes, these changes were done before I added shorthand function. Thanks!